nucypher/blockchain/eth/sol/source/contracts/Issuer.sol

```diff
@@ -98,7 +98,8 @@ abstract contract Issuer is Upgradeable {
         require(localTotalSupply > 0 &&
             _issuanceDecayCoefficient != 0 &&
             _hoursPerPeriod != 0 &&
-            _genesisHoursPerPeriod != 0 &&
+            // prevent overflow during period calculations
+            _genesisHoursPerPeriod >= 10 &&
```

Comment:
😕 ...could you expand on why 10? testing?

Reply:
it comes from minimum seconds per period when overflow occurs (around 6-7 hours), so I took value of 10 hours to have some periods reserve for time traveling (in worst case)